python/tests/tsutil.py

```diff
@@ -539,6 +582,48 @@ def compute_mutation_parent(ts):
     return mutation_parent
 
 
+def compute_mutation_times(ts):
+    """
+    Compute the `time` column of a MutationTable in a TableCollection.
+    Finds the set of mutations on an edge that share a site and spreads
+    the times evenly over the edge.
+
+    :param TreeSequence ts: The tree sequence to compute for.  Need not
+        have a valid mutation time column.
+    """
+    tables = ts.dump_tables()
+    mutations = tables.mutations
+
+    mutations_above_node = collections.defaultdict(list)
+    for mut_idx, mut in enumerate(mutations):
```

Comment:
I think mut_idx is available as ``mut.id``, so can simplify this a bit

Reply:
Sadly it's a `MutationTableRow` so no `id`.

Reply:
ah.